src/Components/Charts/Bar.tsx

```diff
@@ -0,0 +1,387 @@
+import React, { useContext, useEffect, useState } from "react";
+import { ResponsiveBar } from '@nivo/bar'
+import axios from "axios";
+import { AuthContext } from "../../Context/AuthContext";
+
+import { ResponsiveStream } from '@nivo/stream'
+export default function Bar() {
+  const { baseUrl, requstHeaders }: any = useContext(AuthContext);
+  const [contTasks, setContTasks] = useState([]);
```

Comment:
Consider updating the naming convention for the states by using `tasksCount` and `usersCount` instead of `countTasks` and `countUsers`. The rationale behind this adjustment is to follow a more descriptive naming convention, where the prefix "verp" at the beginning of a name typically signifies a function rather than a state variable.